custom_conf.py

```diff
@@ -83,6 +83,10 @@
     # (use an empty value if you don't want to link)
     'mattermost': 'https://chat.canonical.com/canonical/channels/documentation',
 
+    # Change to the Matrix channel you want to link to
+    # (use an empty value if you don't want to link)
+    'matrix': 'https://matrix.to/#/#room-address:ubuntu.com',
```

Comment:
Can you change that to the documentation channel, so we have a valid link in the starter pack documentation?

```suggestion
    'matrix': 'https://matrix.to/#/#documentation:ubuntu.com',
```